### ISSUE TITLE
Missing required accessibility support for tagged PDFs

### ISSUE THREAD

**Issue by normanbrobinson** [jung-kurt/gofpdf#147]
PDF generated without required accessibility features, including tagging metadata.

Example accessibility check using Adobe Acrobat default checker on basic.pdf document:

[basic.pdf.accreport.zip](https://github.com/jung-kurt/gofpdf/files/1502360/basic.pdf.accreport.zip)

**Comment by jung-kurt** [comment]
This is a valuable goal, but after reading section 10.7 of the [PDF Reference](http://digitalgalen.net/Documents/External/pdf_reference_1-7.pdf) I fear achieving it will be a huge task, possibly one that will have to wait until gofpdf supports Unicode natively.

I'm glad to know of the accessibility report.
